src/cfnlint/rules/resources/properties/NumberSize.py

```diff
@@ -72,7 +72,9 @@ def check(self, cfn, properties, specs, path):
                         property_type = specs.get(prop).get('PrimitiveType')
                         value_specs = RESOURCE_SPECS.get(cfn.regions[0]).get(
                             'ValueTypes').get(value_type, {})
-                        if value_specs.get('NumberMax') is not None and value_specs.get('NumberMin') is not None:
+                        # only check NumberMax and NumberMin when AllowedValues don't exist
+                        # otherwise rely on the Allowed Values check to do the validation
+                        if value_specs.get('NumberMax') is not None and value_specs.get('NumberMin') is not None and value_specs.get('AllowedValues') is None:
```

Comment:
Should we just not add `NumberMin`/`NumberMax` when we have the more descriptive `AllowedValues`? Seems simpler

Reply:
Possible I wanted to use it to compare the values of parameter attributes.  If they have MinValue or MaxValue it should be within the bounds of NumberMin and NumberMax or possibly it could be that the MinValue and MaxValue are still valid values?

Reply:
Removed for now.